tests/integration_tests/test_chainermn.py

```diff
@@ -300,7 +302,11 @@ def test_suggest_float_with_step(storage_mode: str, comm: CommunicatorBase) -> N
                 x2 = mn_trial.suggest_float("x", low, high, step=step)
                 assert x1 == x2
 
-                mn_trial.suggest_float("x", low, high)
+                if comm.rank == 0:
+                    with pytest.warns(RuntimeWarning):
+                        mn_trial.suggest_float("x", low, high)
+                else:
+                    mn_trial.suggest_float("x", low, high)
```

Comment:
> But I suppose it doesn't look consistent with other methods, so I'm happy to use `warnings.catch_warnings` instead of the current with `pytest.raises(ValueError)`.

I may not fully understand your opinion, but I also think that `warnings.catch_warnings` is simple .

```suggestion
                with warnings.catch_warnings():
                    warnings.simplefilter("ignore", category=RuntimeWarning)
                    mn_trial.suggest_float("x", low, high)
```